src/helics/ValueFederates.hpp

```diff
@@ -0,0 +1 @@
+

```

Comment:
Another empty file?

Reply:
see above, and ApplicationApi.hpp will include all the files related to the Application API in one shot.